src/lib/routes/client-api/index.ts

```diff
@@ -4,13 +4,20 @@ import MetricsController from '../../features/metrics/instance/metrics.js';
 import RegisterController from '../../features/metrics/instance/register.js';
 import type { IUnleashConfig } from '../../types/index.js';
 import type { IUnleashServices } from '../../services/index.js';
+import { impactRegister } from '../../features/metrics/impact/impact-register.js';
+import { MetricsTranslator } from '../../features/metrics/impact/metrics-translator.js';
 
 export default class ClientApi extends Controller {
     constructor(config: IUnleashConfig, services: IUnleashServices) {
         super(config);
 
+        const metricsTranslator = new MetricsTranslator(impactRegister);
```

Comment:
I'm leaning towards inlining this inside the MetricsController or event better MetricsService if possible. The translator is like a calculator rather than I/O that needs to be swappable.